pkgs/applications/gis/qmapshack/fix-qttranslations-path.patch

```diff
@@ -0,0 +1,74 @@
+diff --git i/src/qmapshack/setup/CAppSetupLinux.cpp w/src/qmapshack/setup/CAppSetupLinux.cpp
+index 63ea06c0..bf6cad20 100644
+--- i/src/qmapshack/setup/CAppSetupLinux.cpp
++++ w/src/qmapshack/setup/CAppSetupLinux.cpp
+@@ -30,7 +30,7 @@ void CAppSetupLinux::initQMapShack()
+     prepareGdal("", "");
+ 
+     // setup translators
+-    QString resourceDir = QLibraryInfo::location(QLibraryInfo::TranslationsPath);
++    QString resourceDir = QLatin1String("@qttranslations@/translations");
```

Comment:
I'm not too familiar with Qt but shouldn't
```suggestion
+    QString resourceDir = QString("@qttranslations@/translations");
```
be enough?

Reply:
`QLatin1String` is more efficient way of specifying constant Latin-1 strings. For more details see https://doc.qt.io/qt-5/qlatin1string.html#details.

Reply:
What about using [`QStringLiteral`](https://doc.qt.io/qt-5/qstring.html#QStringLiteral)?

Reply:
Docs says:
> **Note:** QLatin1String can still be more efficient than QStringLiteral when the string is passed to a function that has an overload taking QLatin1String and this overload avoids conversion to QString.

Anyway, I don't think it's really important in our case.

Reply:
However, you're immediately converting to a `QString` which happens at runtime and is not free when converting from a `QLatin1String`.

Reply:
Well, what about `QLatin1String resourceDir = ...`?